### PR TITLE
adding jerry to acm managedclusters

### PR DESCRIPTION
**Pull request by durandom**
I added jerry to ACM via the ACM web ui and it successfully got imported.

Now following https://www.operate-first.cloud/apps/content/notebooks/onboarding_new_cluster.html#enable-argocd-management-in-acm

Not sure what this does. 🤷 

@HumairAK @tumido can you explain why we need to add it here too? Is this just for state keeping?

### CODE REVIEW

**Comment by durandom** [comment]
I found https://www.operate-first.cloud/apps/content/acm/manage_your_teams_ocp_clusters_via_acm.html which explains the managedclustersets a bit more

File: acm/overlays/moc/infra/managedclusters/jerry.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: cluster.open-cluster-management.io/v1
+kind: ManagedCluster
+metadata:
+  name: jerry
+  labels:
+    cluster.open-cluster-management.io/clusterset: argocd-managed
+    authdeployment: primary
```
Comment (durandom) [inline acm/overlays/moc/infra/managedclusters/jerry.yaml:7]: @HumairAK looking at https://www.operate-first.cloud/apps/content/acm/manage_your_teams_ocp_clusters_via_acm.html#steps 

> However, to configure your cluster to enable auth via our IDP [AuthRealm](https://github.com/operate-first/apps/blob/master/acm/overlays/moc/infra/identitatem/authrealms/primary.yaml) you will need to add the label authdeployment: primary to your clusters’ ManagedCluster resource.

Is this label sufficient to add this cluster to identitatem and configure SSO?

Comment (tumido) [reply acm/overlays/moc/infra/managedclusters/jerry.yaml:7]: Yes, this is sufficient. The rest is taken care of.

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Comment by durandom** [comment]
https://console-openshift-console.apps.jerry.ionos.emea.operate-first.cloud/ doesnt show operate first single sign on. 
The PR was deployed via argocd

**Comment by schwesig** [comment]
fixes https://github.com/operate-first/operations/issues/494

**Pull request merged**